e2e-tests/build

```diff
@@ -8,7 +8,7 @@ test_dir="$(dirname $0)"
 src_dir="$(realpath ${test_dir}/..)"
 IMAGE=$(echo "$IMAGE" | sed -e 's#percona/#perconalab/#')
 
-if [[ ${DOCKER_NOCACHE:-1} == 1 ]]; then
+if [[ ${DOCKER_NOCACHE:-0} == 1 ]]; then
```

Comment:
Why this change?

Reply:
@tplavcic  We have changed it for all our operators to speed up build process . Do you see any issues ?

Reply:
I don't see an issue, was just wondering why didn't we use it from the start and making a change now.